res/controllers/Traktor-Kontrol-S2-MK3-hid-scripts.js

```diff
@@ -752,6 +777,11 @@ TraktorS2MK3.registerOutputPackets = function () {
         this.samplerCallbacks.push(engine.makeConnection("[Sampler" + i + "]", "play", this.samplesOutputHandler));
     }
 
+    this.fxCallbacks = [];
+    for (let group of [ "[Channel1]", "[Channel2]" ]) {
+        this.fxCallbacks.push(engine.makeConnection(`[QuickEffectRack1_${group}]`, "loaded_chain_preset", this.fxOutputHandler));
+    }
+
```

Comment:
For what is `this.fxCallbacks` and `this.samplerCallbacks` needed?
I think the for loop with just engine.makeConnection(... is enough here.